src/Console/Commands/L5BCrud.php

```diff
@@ -397,7 +397,7 @@ protected function frontend_event($key, $name, $stub)
             'namespace'         => '\Events\Frontend\\' . ucfirst(camel_case($key)),
             'event'             => ucfirst(camel_case($key)),
             'model'             => ucfirst(camel_case($key)),
-            'table'             =>  str_plural($key),
+            'table'             => $key,
```

Comment:
'table'             => \Str::plural($key),

could this line use the new Helpers to keep the pluralisation?